dist/waas.v1beta1.yaml

```diff
@@ -0,0 +1,29 @@
+-
+  name: kubectl
+  version: 1.17.17
+  url: https://storage.googleapis.com/kubernetes-release/release/v1.17.17/bin/linux/amd64/kubectl
+-
+  name: sops
+  version: 3.6.1
+  url: https://github.com/mozilla/sops/releases/download/v3.6.1/sops-v3.6.1.linux
+-
+  name: yq
+  version: 3.4.1
+  url: https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+-
+  name: kustomize
+  version: 3.5.4
+  url: https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v3.5.4/kustomize_v3.5.4_linux_amd64.tar.gz
+-
+  name: skaffold
+  version: 1.20.0
+  url: https://github.com/GoogleContainerTools/skaffold/releases/download/v1.20.0/skaffold-linux-amd64
+-
+  name: SopsSecretGenerator
+  version: 1.3.2
+  url: https://github.com/goabout/kustomize-sopssecretgenerator/releases/download/v1.3.2/SopsSecretGenerator_1.3.2_linux_amd64
+  dest: ${XDG_CONFIG_HOME:-$HOME/.config}/kustomize/plugin/goabout.com/v1beta1/sopssecretgenerator
+-
```

Comment:
I believe you can already update this, we pretty much know what the URL for ksecret will be

Reply:
If I update this, it will fail.

Reply:
No, unless you move labels